# Std. metadata for higher-level flow data types
std_msgs/Header header
# Define module specific info
string mission
bool execute_mission